Remember-iOS/Remember-iOS/Source/ViewControllers/CommunityVC.swift

```diff
@@ -7,23 +7,112 @@
 
 import UIKit
 
-class CommunityVC: UIViewController {
+class CommunityVC: BaseViewController {
 
+    // MARK: - UI Component Part
+    
+    @IBOutlet weak var communityTableView: UITableView!
+    @IBOutlet weak var categoryCollectionView: UICollectionView!
+    
+    // MARK: - Vars & Lets Part
+    
+    var communityContentList: [CommunityContentData] = []
+    var categoryList = [String]()
```

Comment:
두개의 선언방식이 달라서 하나로 맞춰주면 좋을 거 같아요!
컨벤션에서는 `communityContentList`가 나타내는 방식을 더 선호합니다.

```suggestion
    var communityContentList: [CommunityContentData] = []
    var categoryList: [String] = []
```

Reply:
오옹 그렇군여..하나로 맞추도록 할게여!